ddp-workspace/projects/ddp-sdk/src/lib/services/logging.service.ts

```diff
@@ -1,18 +1,27 @@
 import { Injectable, Inject } from '@angular/core';
 import { ConfigurationService } from './configuration.service';
 import { LogLevel } from '../models/logLevel';
+import {StackdriverErrorReporterService} from './stackdriverErrorReporter.service';
```

Comment:
whitespace tslint warning:
```suggestion
import { StackdriverErrorReporterService } from './stackdriverErrorReporter.service';
```

Reply:
Fixed.